util/collections.go

```diff
@@ -2,9 +2,21 @@ package util
 
 import (
 	"fmt"
+	"regexp"
 	"strings"
 )
 
+func MatchesAny(regExps []string, s string) bool {
+	if nil != regExps {
+		for i := 0; i < len(regExps); i++ {
```

Comment:
Perhaps this would be better expressed as:
``` for i, regExpr := range regExps {...
```

Reply:
Thanks for the tip. Also, I can probably drop the nil check as I would assume that the nil range will just produce 0 iterations. My Java bg rears its ugly head  :)